pkg/velaql/providers/query/handler_test.go

```diff
@@ -252,7 +252,7 @@ var _ = Describe("Test Query Provider", func() {
 			Expect(err).Should(BeNil())
 			err = prd.ListResourcesInApp(nil, newV, nil)
 			Expect(err).ShouldNot(BeNil())
-			Expect(err.Error()).Should(Equal("var(path=app) not exist"))
+			Expect(err.Error()).Should(Equal("failed to lookup value: var(path=app) not exist"))
```

Comment:
I would suggest to make this `var(path=xxx) not exist` check in a separate typed check function.